SdcCdmLib/README.md

```diff
@@ -0,0 +1,105 @@
+# SdcCdmLib
+
+This directory contains the SDC CDM .NET library, which provides reference implementations for importing and exporting SDC (Structured Data Capture) data. The library consists of three main projects:
+
+- **SdcCdm** - Core library containing the abstract API (`ISdcCdm`) and various importers/exporters
+- **SdcCdmInSqlite** - SQLite-based implementation of the SDC CDM API
+- **SdcCdm.Tests** - Unit tests for the library functionality
+
+## Prerequisites
+
+- .NET 8.0 SDK or later
+- Basic familiarity with .NET development and command-line tools
+
+> **Note**: If you encounter SDK version conflicts, you may need to update the `global.json` file in the repository root to match your installed .NET SDK version.
```

Comment:
The user shouldn't need to change the repository files to get this to work. How do other dotnet projects handle this?